share/completions/eselect.fish

```diff
@@ -33,12 +34,19 @@ function __fish_complete_eselect_action_options
     __fish_eselect_cmd $cmdl[2..-1] | sed -n -re $findregexp | __fish_sgrep '^    --' | sed -re $parseregexp
 end
 
+function __fish_complete_eselect_php_actions
+    set -l sedregexp 's/^  .([0-9]+).   ([A-Za-z0-9\.]+).?+/\1\t\2/'
+
+    if test (__fish_print_cmd_args_without_options)[3] = 'set'
+        eselect php list (__fish_print_cmd_args_without_options)[-1] | sed -r $sedregexp
```

Comment:
>I'm on Archlinux at the moment, that's a bit of the thing. I tend to not want to do make install to get files spread around my filesystem, so I try to run it directly after make which sometimes break ;D

In that case, there's [an AUR package](https://aur.archlinux.org/packages/fish-git/) (which I adapted to my local use to build from my local clone). I've also noticed that it doesn't play great when run directly from the repo directory. I assume it's because it relies on paths supplied by configure, and those are /usr/local and friends by default.

>Installing Gentoo is easy

It's not that I can't follow instructions, it's that

- I find the instructions tedious - all the `mount` boilerplate to setup the chroot (which arch supplies a script for - `arch-chroot`), the bad shell setup (not even bash-completions and only nano) they have....

- I've had weird issues in virtualbox (like the webrsync stuff complaining about a read-only filesystem _after_ it already did half of its stuff)

- It takes ages - I've now been compiling the kernel for half an hour, and it's just gotten to the modules (and configuring the kernel is not something I actually want to do nor would I save much time - though I'm not actually a gentoo user)